tests/pytype_test.py

```diff
@@ -119,16 +168,21 @@ def pytype_test(args):
     while 1:
         while files and len(running_tests) < args.num_parallel:
             f = files.pop()
+            run_cmd = [
+                pytype_exe,
+                '--typeshed-location=%s' % dirs.typeshed,
+                '--module-name=%s' % _get_module_name(f),
+                '--convert-to-pickle=%s' % os.devnull
+            ]
+            if 'stdlib/3' in f:
+                run_cmd += [
+                    '-V 3.6',
+                    '--python_exe=%s' % args.python36_exe
+                ]
```

Comment:
move the definition of run_cmd and the "if" block below, to where it's used? Right now, it's confusing that you're building up this command, but then you're not using it in the "f not in pytype_run" case.

Reply:
done